brainscore/metrics/IndividualCeiling.py

```diff
@@ -0,0 +1,56 @@
+from brainscore.metrics.ceiling import Ceiling
+from brainscore.metrics.transformations import Transformation
+from brainscore.benchmarks._neural_common import average_repetition
+import numpy as np
+import xarray as xr
+
+
+class InterIndividualCrossValidation:  # TODO inheritance Transformation???: InterIndivCrossVal = pipe, Ceiling = apply?
+
+    def __call__(self, assembly):  # TODO add support for list of assemblies

```

Comment:
Would adding support for multiple assemblies make sense? Would most benchmarks be able to cope with assemblies having different sizes in the `presentation ` dimension and being concatenated along the `neuroid `dimension?

Reply:
Yes, `presentation x neuroid` is the default assembly format. But don't worry too much about making it work for everything, let's first get it to run for this particular benchmark and then we can still think about generifying.